### PR TITLE
Made ammendments to code-block ui and language selection

### LINKED ISSUE #827: Code block UI amendment required
Please refer to the following video for more information
- UI issues: https://vimeo.com/842749776/8e3360680b

@AbhayVAshokan _a
cc: @yedhink 

---

**Action items**

- [x] Saving a codeblock converts certain characters to `&lt;`, `&gt;`, etc.
- [x] Remove text wrapping from the CodeBlocks in edit mode.
- [x] Fix the position of copy button when the content scrolls.
- [x] Verify the behaviour of dropdowns during scroll.
- [x] Remove `auto` highlight and make `plaintext` by default.

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #827 

**Description**

- Fixed: styling of code-block
- Added: "plaintext" as default
- Fixed: issues with parsed coodeblock in EditorContent

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

@AbhayVAshokan _a

### CODE REVIEW

File: src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx [hunk 1]
```diff
@@ -28,6 +28,10 @@ const CodeBlockComponent = ({ node, updateAttributes }) => {
     setKeyword("");
   };
 
+  useEffect(() => {
+    updateAttributes({ language: "plaintext" });
+  }, []);
```
Comment (AbhayVAshokan) [inline src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx:33]: https://vimeo.com/844942928/16b5819463

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please review the new changes.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a we need strategy="fixed" for the dropdown so that the dropdown is visible:
image

Currently this is cut off within the boundaries of the codeblock.

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please take a look at the updated  Ui, I have reverted the changes to dropdown strategy.

File: src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx
Comment (AbhayVAshokan) [inline src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx:1]: - [ ] Remove `closeOnOutsideClick={false}` from the dropdown component. It is not user friendly.
- [ ] Remove auto from the list of available languages.
image

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul _a please merge yourselves after this change.

**Pull request merged**